addons/website/static/src/interactions/carousel_slider.edit.js

```diff
@@ -0,0 +1,65 @@
+import { registry } from "@web/core/registry";
+import { Interaction } from "./carousel_slider";
+
+export class CarouselSliderEdit extends CarouselSlider {
+    // Pause carousel in edit mode.
+    carouselOptions = {ride: false, pause: true};
+
+    setup() {
+        Object.apply(this.dynamicContent, {
+            "_root": {
+                "t-on-contentChanged": this.onContentChanged,
+            },
+        });
+        // Only for carousels having the `Carousel` and `CarouselItem` options
+        // (i.e. matching the `section > .carousel` selector).
+        if (this.el.matches("section > .carousel")) {
+            // TODO Handle translation mode.
+            // && !this.options.wysiwyg.options.enableTranslation) {
+            Object.apply(this.dynamicContent, {
+                ".carousel-control-prev, .carousel-control-next": {
```

Comment:
would it be wrong to always do it, and not only if we have a "section > .carousel"?